core/templates/components/question-directives/questions-list/questions-list.component.ts

```diff
@@ -497,6 +497,8 @@ angular.module('oppia').component('questionsList', {
         ctrl.updateSkillLinkage();
       };
       ctrl.isQuestionSavable = function() {
+        $rootScope.$applyAsync();
+
```

Comment:
How many times is this invoked? Why are we forcing a digest loop inside a validation method -- it does not seem to modify any data?

Reply:
after solving e2e no need to use this line here so this line is deleted now.

Reply:
Sounds good, thanks!